at.vintagestory.VintageStory.yaml

```diff
@@ -0,0 +1,62 @@
+app-id: at.vintagestory.VintageStory
+runtime: org.freedesktop.Platform
+runtime-version: '20.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+    - org.freedesktop.Sdk.Extension.mono6
+command: /app/bin/vintagestory
+finish-args:
+    - --socket=x11
+    - --share=ipc
+    - --device=dri
+    - --share=network
+    - --socket=pulseaudio
+    - --env=MONO_PATH=/app/lib/mono/4.5
+modules:
+    - name: mono
+      buildsystem: simple
+      build-commands:
+          - /usr/lib/sdk/mono6/install.sh
+          - cp /usr/lib/sdk/mono6/lib/mono/4.5/cert-sync.exe /app/lib/mono/4.5/
+    - name: libgdiplus
+      sources:
+          - type: archive
+            url: https://download.mono-project.com/sources/libgdiplus/libgdiplus0-6.0.5.tar.gz
+            sha256: b81e4e5cc3e4831b2945de08bef26eb1bdcd795aeaf8f971b221c51213a025ef
+    - name: VintageStory
+      buildsystem: simple
+      build-commands:
+          - install -Dpt /app/bin vintagestory apply_extra
+          - install -Dpt /app/share/icons/hicolor/512x512/apps at.vintagestory.VintageStory.png
+          - install -Dpt /app/share/metainfo at.vintagestory.VintageStory.metainfo.xml
+          - desktop-file-install --dir=/app/share/applications at.vintagestory.VintageStory.desktop
+      sources:
+          - type: script
+            dest-filename: vintagestory
+            commands:
+                # cert-sync is needed to update mono's certificates, or
+                # communication with Vintage Story's auth server may fail
+                - /app/bin/mono ${MONO_PATH}/cert-sync.exe --quiet --user /etc/ssl/certs/ca-certificates.crt
+                - exec /app/bin/mono /app/extra/vintagestory/Vintagestory.exe
+          - type: file
+            path: at.vintagestory.VintageStory.desktop
+          - type: file
+            # from converting Vintage Story's assets/gameicon.xpm to png
+            path: at.vintagestory.VintageStory.png
```

Comment:
The image says it's empty, is that expected?

Reply:
I don't know why the github description says it is empty. When I click on the dots > View file, the image shows.